packages/in_app_purchase/in_app_purchase_android/lib/src/billing_client_wrappers/purchase_wrapper.dart

```diff
@@ -33,15 +33,16 @@ class PurchaseWrapper {
     required this.purchaseTime,
     required this.purchaseToken,
     required this.signature,
-    required this.sku,
+    @Deprecated('Use skus instead') String? sku,
```

Comment:
I think we need to update the shared plugin to use the new field instead. Should be in a new PR after this is published